### PR TITLE
[MIG] purchase_requisition_tier_validation: Migration to 16.0

### PR DESCRIPTION
**Pull request by dzungtran89**

### CODE REVIEW

**Comment by rven** [comment]
Isn't this already migrated with https://github.com/OCA/purchase-workflow/pull/1671?

**Comment by dzungtran89** [comment]
> Isn't this already migrated with https://github.com/OCA/purchase-workflow/pull/1671?

No, they are two diff modules

**Review by yankinmax — APPROVED** [review]

**Comment by yankinmax** [comment]
@dzungtran89 pls rebase on latest 16.0 updates. pre-commit should be fixed now

File: purchase_requisition_tier_validation/tests/test_purchase_requisition_tier_validation.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestPurchaseRequisitionTierValidation(common.TransactionCase):
+    def setUp(self):
```
Comment (leemannd) [inline purchase_requisition_tier_validation/tests/test_purchase_requisition_tier_validation.py]:
```
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.tier_definition = cls.env["tier.definition"]
```

**Review by leemannd — APPROVED** [review]
LGTM
Minor comment, test could be moved to setupClass

**Review by cyrilmanuel — APPROVED** [review]
LGTM : 
pre-commit can be resolve regarding this : 

https://github.com/OCA/oca-addons-repo-template/issues/185

create new PR:  checkout the branch (eg: 16.0) fork it and run copier -f update

**Comment by dzungtran89** [comment]
Good day, thank you for the review, the PR has been updated

**Review by cyrilmanuel — APPROVED** [review]

**Pull request merged**